tests/manual/events/KeyCharacterCombinationTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.control.TextArea;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCombination;
+import javafx.scene.input.KeyCharacterCombination;
+import javafx.scene.input.KeyEvent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class KeyCharacterCombinationTest extends Application {
+    private final TextArea typingArea = new TextArea("");
+    private KeyEvent lastPressed = null;
+
+    public static void main(String[] args) {
+        Application.launch(args);
+    }
+
+    @Override
+    public void start(Stage stage) {
```

Comment:
I started the test app but was unsure how to test it. Maybe add a header text that explains how a successful test should go.

Reply:
There's now a better test app in tests/manual/events/KeyboardTest.java. With KeyCharacterCombinationTest the human tester has to press a bunch of keys. KeyboardTest.java uses a Robot to automate the process. The downside is that KeyboardTest.java relies on working Robot code so it can't really be used on Linux until #718 is integrated. I'll update the comments on this in the interim.